### PR TITLE
Added release notes

### PR DESCRIPTION
**Pull request by pranchalsomani**

### CODE REVIEW

File: ReleaseNotes/RN230316.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ * Added [Bulk Remediation Script](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1) for the control [Azure_SQLDatabase_AuthZ_Use_AAD_Only](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Readme.md#39-Azure_SQLDatabase_AuthZ_Use_AAD_Only)
+ * Added [Bulk Remediation Script](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1) for the control [Azure_AutomationAccounts_DP_Encrypt_Variables](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Readme.md#40-Azure_AutomationAccounts_DP_Encrypt_Variables)
+
+## Tenant Scanner Framework Updates
+* Updated the target framework for all AzTS components to **net6.0** and function runtime version to **V4** in the entire AzTS solution as netcore3.1 is out of support.
```
Comment (ZiaAhmad-PM) [inline ReleaseNotes/RN230316.md:37]: as **netcore 3.1** is out of support.

Pls. link phrase "out of support" with the Microsoft notice if available.

File: ReleaseNotes/RN230316.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+
+## Other Improvements
+* **AzTS UI** : Added new feature - Control remediation using scripts download from AzTS UI.
```
Comment (ZiaAhmad-PM) [inline ReleaseNotes/RN230316.md:41]: Is this feature available for outside users. Could we link the GitHub documentation? 

**AzTS UI** : Added new feature - Control remediation using scripts download from AzTS UI. Read **here** for more details.

File: ReleaseNotes/RN230316.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+*  **AzTS UI** :  Introduced a single side panel view on clicking over control name to display control and control evaluation related details.
+* **Policy** : Added new [Policy](https://github.com/azsk/AzTS-docs/blob/main/Policies/Readme.md) for following controls.
```
Comment (ZiaAhmad-PM) [inline ReleaseNotes/RN230316.md:43]: Added new audit policies for the following security controls.

File: ReleaseNotes/RN230316.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    *  [Azure_KubernetesService_AuthN_Enabled_AAD](https://github.com/azsk/AzTS-docs/blob/main/Policies/KubernetesService/Azure_KubernetesService_AuthN_Enabled_AAD/Readme.md)
+    *  [Azure_Subscription_Config_MDC_Setup_SecurityContacts](https://github.com/azsk/AzTS-docs/blob/main/Policies/Subscription/Azure_Subscription_Config_MDC_Setup_SecurityContacts/Readme.md)
+    *  [Azure_APIManagement_AuthN_Use_AAD_for_Client_AuthN](https://github.com/azsk/AzTS-docs/blob/main/Policies/APIManagement/Azure_APIManagement_AuthN_Use_AAD_for_Client_AuthN/Readme.md)
+    *  [Azure_KeyVault_NetSec_Disable_Public_Network_Access](https://github.com/azsk/AzTS-docs/blob/main/Policies/KeyVault/Azure_KeyVault_NetSec_Disable_Public_Network_Access/Readme.md)
+    *  [Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server](https://github.com/azsk/AzTS-docs/blob/main/Policies/SQLServer/Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server/Readme.md)
+
+* **Policy** : Updated [Policy](https://github.com/azsk/AzTS-docs/blob/main/Policies/Readme.md) for following controls .
```
Comment (ZiaAhmad-PM) [inline ReleaseNotes/RN230316.md:51]: Updated audit policies for the following security controls.

**Review by ZiaAhmad-PM — CHANGES_REQUESTED** [review]

File: ReleaseNotes/RN230316.md (hunk 4 above)
Comment (pranchalsomani) [reply ReleaseNotes/RN230316.md:51]: Done

File: ReleaseNotes/RN230316.md (hunk 3 above)
Comment (pranchalsomani) [reply ReleaseNotes/RN230316.md:43]: Done

File: ReleaseNotes/RN230316.md (hunk 2 above)
Comment (pranchalsomani) [reply ReleaseNotes/RN230316.md:41]: Yes. Linked

File: ReleaseNotes/RN230316.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+ * Added [Bulk Remediation Script](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1) for the control [Azure_AutomationAccounts_DP_Encrypt_Variables](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Readme.md#40-Azure_AutomationAccounts_DP_Encrypt_Variables)
+
+## Tenant Scanner Framework Updates
+* Updated the target framework for all AzTS components to **net6.0** and function runtime version to **V4** in the entire AzTS solution as netcore3.1 is [out of support](https://devblogs.microsoft.com/dotnet/net-core-3-1-will-reach-end-of-support-on-december-13-2022/).
+
+
+## Other Improvements
+* **AzTS UI** : Added new feature - Control remediation using scripts download from AzTS UI.Read [here](https://github.com/azsk/AzTS-docs/blob/main/04-Addressing%20control%20failure/RemediationThroughScriptsUsingUI.md) for more details.
```
Comment (ZiaAhmad-PM) [inline ReleaseNotes/RN230316.md:41]: A space to be added before "Read here"

Comment (pranchalsomani) [reply ReleaseNotes/RN230316.md:41]: Done

**Review by ZiaAhmad-PM — APPROVED** [review]
Approved

**Review by Aboli-msft — APPROVED** [review]

**Pull request merged**